src/ProgressOnderwijsUtils/SingleSignOn/SsoProcessor.cs

```diff
@@ -43,7 +43,7 @@ XElement ToXml()
                     new XAttribute("Version", "2.0"),
                     new XAttribute("IssueInstant", DateTime.UtcNow),
                     new XAttribute("Destination", Destination),
-                    new XAttribute("ForceAuthn", "false"),
+                    new XAttribute("ForceAuthn", "true"),

```

Comment:
kan dit niet een parameter zijn? Dit lijkt me soms wel (bv pnet) en soms niet gewenst (bv verkiezingen/p3w)

Reply:
Uiteraard kan dat, maar het vereist wel een enigszins grote herschrijfactie om überhaupt parameters mee te kunnen geven.

Reply:
En hij zou bij de verkiezingen nooit uit moeten maken, want we hebben onze eigen cookie; dus je komt ueberhaupt alleen bij de provider als het "lang" geleden is dat je ingelogd was; idem p3w.

Reply:
Nee, maar je denkt verkeerdom. Iemand zit op mijn.hanze.nl, klikt op verkiezingen, en moet nog een keer inloggen. Niet: iemand zit in verkiezingen en gaat nog een keer naar verkiezingen. Het idee van singlesingnon is juist dat je niet per app inlogt.

Reply:
Voor de verkiezingen is dit niet handig nee.

Reply:
Dus toch wel parameteriseren?

Reply:
Ja